components/button/test/button-icon.visual-diff.js

```diff
@@ -35,8 +34,8 @@ describe('d2l-button-icon', () => {
 					const selector = `#${entry.category}`;
 
 					if (name === 'hover') await page.hover(selector);
-					else if (name === 'focus') await page.$eval(selector, (elem) => forceFocusVisible(elem));
-					else if (name === 'click') await page.$eval(selector, (elem) => elem.focus());
+					else if (name === 'focus') await focusWithKeyboard(page, selector);
+					else if (name === 'click') await focusWithMouse(page, selector);
```

Comment:
In hindsight, the "click" tests aren't named super well for what they were capturing. I know you probably don't want to cause a bunch of diff changes though.